omero/util_scripts/Dataset_To_Plate.py

```diff
@@ -115,10 +116,17 @@ def dataset_to_plate(conn, script_params, dataset_id, screen):
     if remove_dataset:
         remove_from = dataset
 
-    for image in images:
-        added_count = add_image_to_plate(conn, image,
-                                         plate.getId().getValue(),
-                                         col, row, remove_from)
+    images_per_well = script_params["Images_Per_Well"]
+    image_index = 0
+
+    while image_index < len(images):
+
+        well_images = images[image_index: image_index + images_per_well]
```

Comment:
So this means only the following orders should be supported: `[row, column, well sample]` or `[column, row, well sample]`. Does that match the use case reported in the original thread or is it possible we might have to support more combinations?